public/pages/utils/constants.ts

```diff
@@ -20,6 +20,7 @@ export enum DETECTOR_STATE_COLOR {
   DISABLED = 'subdued',
   INIT = '#0000cc',
```

Comment:
can you try 'primary' for INIT? I recalled 'primary' is blue color as well.

Reply:
I see, yes. Will change.